pkg/dock/dock.go

```diff
@@ -585,6 +584,18 @@ func (ds *dockServer) DeleteFileShareSnapshot(ctx context.Context, opt *pb.Delet
 		log.Error("error occurred in dock module when delete snapshot:", err)
 		return pb.GenericResponseError(err), err
 	}
-	// TODO: maybe need to update status in DB.
+
 	return pb.GenericResponseResult(nil), nil
 }
+
+// GetMetrics method is only defined to make ProvisioinDock service consistent with
+// ProvisionController service, so this method is not allowed to be called.
+func (ds *dockServer) GetMetrics(context.Context, *pb.GetMetricsOpts) (*pb.GenericResponse, error) {
```

Comment:
Why do we need these two functions just to send the message that it is not implemented?

Reply:
If you look at [comments](https://github.com/opensds/opensds/pull/888#discussion_r294332438) from xing, we came to conclusion that we should keep thin-opensds as consistent as possible with master branch.